sys/linux/dev_dri.txt

```diff
@@ -170,14 +171,14 @@ ioctl$DRM_IOCTL_MODE_GETCONNECTOR(fd fd_dri, cmd const[DRM_IOCTL_MODE_GETCONNECT
 
 drm_mode_get_connector {
 # Everything except for connector_id are outputs.
-	encoders_ptr		ptr64[out, array[int32]]
+	encoders_ptr		ptr64[out, array[drm_encoder_id]]
 	modes_ptr		ptr64[out, array[drm_mode_modeinfo]]
 	props_ptr		ptr64[out, array[int32]]
 	prop_values_ptr		ptr64[out, array[int32]]
 	count_modes		len[modes_ptr, int32]
 	count_props		len[props_ptr, int32]
 	count_encoders		len[encoders_ptr, int32]
-	encoder_id		const[0, int32]
+	encoder_id		drm_encoder_id
```

Comment:
`ioctl$DRM_IOCTL_MODE_GETCONNECTOR` seems to have its argument marked as `arg ptr[in, ...]`. It might need to be changed to `inout` with `encoder_id` annotated as `(out)`, and `connector_id` as `(in)`.

Reply:
Do we need to mark `count_modes` (and friends) as `(in)` as well then?

Reply:
I don't think so-- looking at the existing descriptions inside sys/linux/, only a few of the len[...] fields seem to be explicitly annotated as (in).

Reply:
ok. I'll just leave this then.